src/api/FetchData.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+const getFetch = async (object: string, params: string) => {
+  const res = await axios.get(import.meta.env.VITE_API_BASE_URL + object + params);
+  const data = res.data;
+  return data;
+};
+
+const postFetch = async (object: string, body: Record<string, string> | null) => {
+  const res = await axios.post(process.env.VITE_API_BASE_URL + object, body);
+  const data = res.data;
+  return data;
+};
+type httpTypes = 'GET' | 'POST';
+
+export const ApiClient = async (
```

Comment:
잘못 전달 된 것으로 보이네요.. 타입은 PascalCase, 아래 ApiClient는 camelCase로 작성해주면 좋아보여요
```suggestion
type HttpTypes = 'GET' | 'POST';

export const apiClient = async (
```